### ISSUE TITLE
lottie-flutter does not render animation properly

### ISSUE THREAD

**Issue by wurikiji** [dnfield/lottie-flutter#29]
The end of the animation should be this.
![KakaoTalk_Photo_2019-05-16-00-40-03](https://user-images.githubusercontent.com/5652744/57789098-396e8700-7773-11e9-85df-4813d2dd5a3e.jpeg)

But, lottie-flutter renders like this on both android and ios. 
![579626707 27](https://user-images.githubusercontent.com/5652744/57789134-45f2df80-7773-11e9-852f-d09c3c3370ea.png)

You can download the lottie json file. 
[splash.zip](https://github.com/dnfield/lottie-flutter/files/3183324/splash.zip)

## Flutter doctor

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.5.9-pre.174, on Mac OS X 10.14.5 18F132, locale ko-KR)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
[✓] Android Studio (version 3.4)
[✓] IntelliJ IDEA Ultimate Edition (version 2019.1.2)
[✓] VS Code (version 1.33.1)
[✓] Connected device (1 available)

**Comment by wurikiji** [comment]
And also, the version of `lottie_flutter` on packages site is `v0.2.0`. Could you publish the updated version to the pub packages ?

**Comment by wurikiji** [comment]
To find the rotation gap between correct version and lottie_flutter version, I manually modify the `splash.json` that I linked. The upper left arc need to be rotated by 108 degree, the upper right arc nee d to be roated by 245 degree, and the bottom right arc need to rotated by 225 degree. The bottom left arc are correct.

**Comment by dnfield** [comment]
I don't have a great answer for you on this - this plugin is currently in a bad state.  I've started a complete rewrite of it, but don't yet have an ETA for when I can get that done (I have a few other higher priority items I have to get to first).  I'm hoping to make more progress on it in the next month or so.

I've tried to clean the current state up a few times and always end up breaking things - which is why I haven't published a newer version.
